sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/responses/SubscribeVehicleDataResponseTest.java

```diff
@@ -193,6 +197,7 @@ public void testRpcValues () {
         assertNull(Test.NULL, msg.getEmergencyEvent());
         assertNull(Test.NULL, msg.getClusterModeStatus());
         assertNull(Test.NULL, msg.getMyKey());
+        assertNull(Test.NULL, msg.getTurnSignal());
```

Comment:
Spacing in this file is little strange. It doesn't look like it is introduced in this PR though.